src/components/Organism/Policy.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import Link from "@/components/Atom/Link";
+import media from "@/util/breakpoint";
+
+const Wrapper = styled.ul`
+  width: 100%;
+  margin-bottom: 60px;
+
+  ${media.lessThan("lg")`
+    padding: 0 3vw;
+  `}
+`;
+const List = styled.li`
+  width: 100%;
+  &:nth-of-type(n + 2) {
+    margin-top: 40px;
+  }ﾆ
```

Comment:
```suggestion
  }
```

謎のハイフンが入っているので消しましょう